src/OrbitGl/TimelineUiTest.cpp

```diff
@@ -186,4 +187,28 @@ TEST(TimelineUi, Draw) {
   timeline_ui_test.TestDraw(kMinTick, kMaxTick, /*mouse_tick=*/std::nullopt);
 }
 
+TEST(TimelineUi, OnMouseWheel) {
+  using ::testing::_;
+  using ::testing::Exactly;
+  using ::testing::Return;
+
+  int width = TimelineUiTest::kBigScreenPixelsWidth;
+  MockTimelineInfo mock_timeline_info;
+  mock_timeline_info.SetWorldWidth(width);
+
+  TimeGraphLayout layout;
+  Viewport viewport(width, 0);
+  TimelineUiTest timeline_ui_test(&mock_timeline_info, &viewport, &layout);
+  const Vec2 kMousePosInsideTimeline = timeline_ui_test.GetPos();
```

Comment:
this is using the constant syntax, but it's not a constant. mouse_pos_inside_timeline?

Reply:
sorry, this is constant but not related with Mouse. I should erase Mouse from the name.

Reply:
I think the point here is more to not use camelCase syntax - and actually I'm not sure if https://google.github.io/styleguide/cppguide.html#Constant_Names applies here. Technically this is a `const` variable whose "value is fixed for the duration of the program", but I also agree that it's weird and I'd only use this naming convention for variables fixed at compile time.

Reply:
I renamed to pos_inside_timeline